### ISSUE TITLE
test3

### ISSUE THREAD

**Issue by Methuselah96** [Methuselah96/immutable-js#3]
more stuff

**Comment by mcclure** [comment]
Looks like you imported the issues in reverse chronological order (not a problem) but also duplicated some of the issues (maybe a problem)

**Comment by Methuselah96** [comment]
@mcclure Yes, thank you. I was just testing stuff out and mentioned people by mistake.
